src/lib/y2storage/lvm_lv.rb

```diff
@@ -179,7 +186,11 @@ def overcommitted?
     # closest valid (i.e. divisible by the extent size) value, rounding down if
     # needed.
     #
-    # @param new_size [DiskSize] temptative new size of the volume, take into
+    # The resulting size is also rounded down according to the number of stripes,
+    # except when setting the minimum size. Note that lvcreate command will round
```

Comment:
NP: I guess that "lvcreate will round it up again" only applies if the size was set to the minimum. I'm not sure if it's fully obvious from the current wording.